src/main/java/com/cdancy/bitbucket/rest/features/PullRequestApi.java

```diff
@@ -102,4 +102,13 @@ PullRequest reopen(@PathParam("project") String project,
                        @PathParam("repo") String repo,
                        @PathParam("pullRequestId") int pullRequestId,
                        @QueryParam("version") int version);
+
+    @Named("pull-request:changes")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/{project}/repos/{repo}/pull-requests/{pullRequestId}/changes")
+    @Fallback(PullRequestOnError.class)
+    @GET
+    PagedResponse<Change> changes(@PathParam("project") String project,
```

Comment:
***PagedResponse*** is generic and lots of resources are paged.
***Path*** and ***Changes*** are both shared with the commits resource.

Do you still think they should be put under pullrequest?